pkgs/os-specific/linux/waydroid/default.nix

```diff
@@ -56,8 +58,9 @@ python3Packages.buildPythonApplication rec {
   preFixup = ''
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
 
+    patchShebangs $out/lib/waydroid/data/scripts
     wrapProgram $out/lib/waydroid/data/scripts/waydroid-net.sh \
-       --prefix PATH ":" ${lib.makeBinPath [ dnsmasq getent iproute2 nftables ]}
+      --prefix PATH ":" ${lib.makeBinPath [ dnsmasq getent iproute2 iptables nftables ]}
```

Comment:
Can we remove iproute2 or nftables now?

Reply:
iproute2 is always needed, see https://github.com/waydroid/waydroid/blob/main/data/scripts/waydroid-net.sh#L64. But I thought we can remove nft.

Reply:
I removed nftables.